tests/integration/class-with-method/input/index.rst

```diff
@@ -0,0 +1,15 @@
+=================================
+PHP Class with implicit namespace
```

Comment:
```suggestion
PHP Class with explicit namespace
```

Would name it explicit, as you give it explicitly as FQCN. Or use "PHP Class with FQCN". But this is not a blocker.